go.mod

```diff
@@ -3,3 +3,20 @@ module github.com/hajimehoshi/go-mp3
 go 1.14
 
 require github.com/hajimehoshi/oto/v2 v2.3.1
+
+// Fixed issues with internal module import paths for forked repositories.
+replace github.com/hajimehoshi/go-mp3/internal/bits => ./internal/bits
+
+replace github.com/hajimehoshi/go-mp3/internal/consts => ./internal/consts
+
+replace github.com/hajimehoshi/go-mp3/internal/frame => ./internal/frame
+
+replace github.com/hajimehoshi/go-mp3/internal/frameheader => ./internal/frameheader
+
+replace github.com/hajimehoshi/go-mp3/internal/huffman => ./internal/huffman
+
+replace github.com/hajimehoshi/go-mp3/internal/imdct => ./internal/imdct
+
+replace github.com/hajimehoshi/go-mp3/internal/maindata => ./internal/maindata
+
+replace github.com/hajimehoshi/go-mp3/internal/sideinfo => ./internal/sideinfo
```

Comment:
?

Reply:
Because relative import paths cannot be used in Go, if you fork a Go project that has internal modules, the Go tools will report messages such as "use of internal package X not allowed". Ultimately, newer versions of Go modules deal with this problem, but the warning messages still appear. The proposed solution does not break dependencies and avoids alarming warnings.

Reply:
> Because relative import paths cannot be used in Go

The solution is to avoid relative import paths, not adding `replace` directives.

Reply:
If you want to use your local fork, you can use https://go.dev/doc/tutorial/workspaces

Reply:
Wow, I didn't know that. Thank you very much! :)